### DISCUSSION TITLE
Upgrading from v 1.6 to 2.0 problems

### DISCUSSION THREAD

**Discussion by PRY2022** [artilleryio/artillery#1865] [Artillery]
Hi, I'm upgrading from 1.6 Artillery.IO to 2.0 and my flow is not working as expected.

I'm making a POST request to some url(1) and receive a result page(2) where I need to capture some fields and make a second POST request(3).
 
The main problem is that result page(2) represents a page with a hidden form and a js script that autosubmits this form upon loading.

(function () {
    document.forms[0].submit();
})();

In 1.6 version scenario all works fine, it simply stops on the first POST result, I can capture fields and perform the second POST.
2.0 version executes the above js script immediately, redirects and throws 405 error that POST is not allowed. So I'm not able to capture or receive necessary fields for next steps. 
How this behavior could be resolved?  Is there an option to disable js execution?

**Comment by hassy** [comment]
hi @PRY2022 👋 

Artillery's built-in HTTP engine does not execute Javascript code on web pages. Are you running tests with Playwright?

**Reply by PRY2022** [reply]
Nope, simply starting manually in node.js. 
I've recently found a thread that in 1.7 that cookie treatment is different since this version:
https://github.com/artilleryio/artillery/issues/1035
and adding 
 defaults: #
    cookie: #
to config made it possible to deal with GET requests in 2.0 but POST request is failing and I'm not that competent to understand why.
